app/Http/Controllers/DoctorController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\StoreDoctorRequest;
+use Illuminate\Http\Request;
+
+class DoctorController extends Controller
+{
```

Comment:
Place the doctor registration function here

Reply:
done